es-ds-docs/pages/molecules/slider.vue

```diff
@@ -0,0 +1,66 @@
+<script setup>
+const model = ref(null);
+const min = ref(0);
+const max = ref(1000);
+const step = ref(50);
+const ariaLabel = ref('Sample aria-label');
+
+const { $prism } = useNuxtApp();
+const compCode = ref('');
+const docCode = ref('');
+
+if ($prism) {
```

Comment:
Can we apply the prism fix here that was applied to other components? (wrapping the if block in an async function inside onMounted)